setup.py

```diff
@@ -32,5 +32,6 @@
         "Programming Language :: Python :: 3.5",
         "Programming Language :: Python :: 3.6",
         "Programming Language :: Python :: 3.7",
+	"Programming Language :: Python :: 3.8",
```

Comment:
Seems to be some inconsistent spacing here.

Reply:
Hehe 😅 thanks correcting. When editing with VIM